projects/VRDR.CLI/Program.cs

```diff
@@ -638,7 +638,7 @@ static int Main(string[] args)
                 DeathRecord d2 = new DeathRecord(d1.ToJSON());
                 DeathRecord d3 = new DeathRecord();
                 List<PropertyInfo> properties = typeof(DeathRecord).GetProperties().ToList();
-                HashSet<string> skipPropertyNames = new HashSet<string>() { "CausesOfDeath", "AgeAtDeathYears", "AgeAtDeathMonths", "AgeAtDeathDays", "AgeAtDeathHours", "AgeAtDeathMinutes" };
+                HashSet<string> skipPropertyNames = new HashSet<string>() { "UsualOccupationCoded", "UsualIndustryCoded", "DeathRecordIdentifier", "CausesOfDeath", "AgeAtDeathYears", "AgeAtDeathMonths", "AgeAtDeathDays", "AgeAtDeathHours", "AgeAtDeathMinutes" };

```

Comment:
Why was DeathRecordIdenitifier added to the skip list? Is there a reason why it can't be in a round trip or is it a bug?

Reply:
Per @pkmitre :  DeathRecordIdentifier is odd because it’s a derived field, and so should generally not be set directly (it has a private setter method).

It was failing in round trip tests of the CLI:
 I can convert from json without incident or data loss.
The roundtrip from xml fails.
But this roundtrip-all fails on the first field (Identifier):
[***** MISMATCH *****] "2018MA000001" (property: DeathRecordIdentifier) does not equal "2018MA000000" 1:"2018MA000001" 2:"2018MA000001" 3:"2018MA000000"
#1 is original
#2 is the json
#3 is the field created programmatically as you describe.

@pkmitre suggested skipping this field.